MediaManager/Platforms/Apple/AppleMediaManagerBase.cs

```diff
@@ -202,16 +193,48 @@ public override Task Play()
             return Task.CompletedTask;
         }
 
-        public override Task PlayNext()
+        public override Task<bool> PlayNext()
         {
-            NativeMediaPlayer.Player.AdvanceToNextItem();
-            return Task.CompletedTask;
+            // If we repeat just the single media item, we do that first
+            if (MediaPlayer.RepeatMode == RepeatMode.One)
```

Comment:
Seems like logic to check repeat or shuffle should be central.

Reply:
Done!